testing_exercise/testingSpec.js

```diff
@@ -1,3 +1,49 @@
 var expect = chai.expect;
 
-// WRITE YOUR TESTS HERE!
\ No newline at end of file
+// WRITE YOUR TESTS HERE!
+describe("Tests", function(){
+
+	it("replaces lowercase", function(){
+		expect(replaceWith("awesome", "e", "z")).to.be.equal("awzsomz");
+	});
+
+	it("replaces case sensitive", function(){
+		expect(replaceWith("Foo", "F", "B")).to.be.equal("Boo");
+	});
+
```

Comment:
I'd also write a test to make sure that if the cases DON'T match, no replacement occurs, e.g. `replaceWith("HELLO everyone", "O", "A"); // "HELLA everyone"`